src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -1,11 +1,51 @@
 <template>
     <div>
-        <!-- TODO:PC -->
+        <ul class="nav nav-tabs">
+            <li v-for="plotName of plotNames" :key="plotName">
+                <a class="nav-link"
+                   :class="selectedPlot === plotName ? 'active': ''"
+                   v-translate="plotName"
+                   @click="switchTab(plotName)"></a>
+            </li>
+        </ul>
+        <div class="mt-2">
+            <PlotControlSet :active-plot="selectedPlot"/>
+            <h4 v-translate="'filters'"/>
+            <FilterSet :active-plot="selectedPlot"/>
+        </div>
     </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import {computed, defineComponent, ref} from "vue";
+import FilterSet from "../plots/FilterSet.vue";
+import PlotControlSet from "../plots/PlotControlSet.vue";
+import {PlotName} from "../../store/plotSelections/plotSelections";
+import {useStore} from "vuex";
 
-export default defineComponent({})
+export default defineComponent({
+
+    setup() {
+        const store = useStore()
+
+        const plotNames = computed(() => {
+            return Object.keys(store.state.plotSelections) as PlotName[]
```

Comment:
Can just get these from the array defined in plotSelections.ts, there's an array of what you're looking for, i don't think this needs to be computed